eventlog/tests.py

```diff
@@ -0,0 +1,71 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.utils import timezone
+
+from roster.models import Site, Period, ClusiveUser
+from eventlog.models import Session, Event
+
+# TODO copied from roster tests. Is there a central place that some of this setup could be done?
```

Comment:
The `shared` directory that currently contains only the templates and static files could be used for this purpose. 

Alternately, it would be possible to refactor the Roster test functions so that they can suit both tests, then import the test functions from Roster like so: `from roster.tests import set_up_test_sites, set_up_test_periods, set_up_test_users` - this might be more appropriate, since these are related to creating test versions of the Roster model objects.

Reply:
Will import from roster.tests.